### PR TITLE
add tooltips to "kubelet" term definition

### PR DESCRIPTION
**Pull request by kirisakow**

### CODE REVIEW

**Comment by kundan2707** [comment]
@kirisakow  please sign CLA first

File: content/en/docs/reference/glossary/mirror-pod.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ aka:
...
 
 <!--more--> 
 
-When the kubelet finds a static pod in its configuration, it automatically tries to
+When the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} finds a static pod in its configuration, it automatically tries to
 create a Pod object on the Kubernetes API server for it. This means that the pod
 will be visible on the API server, but cannot be controlled from there.
 
-(For example, removing a mirror pod will not stop the kubelet daemon from running it).
+(For example, removing a mirror pod will not stop the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} daemon from running it).
```
Comment (sftim) [inline content/en/docs/reference/glossary/mirror-pod.md:21]: Better to leave these as they are. We only use a tooltip for the _first_ relevant use of a term.

File: content/en/docs/reference/glossary/node-pressure-eviction.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ pods to reclaim resources on nodes.
 
 <!--more-->
 
-The kubelet monitors resources like CPU, memory, disk space, and filesystem 
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} monitors resources like CPU, memory, disk space, and filesystem 
 inodes on your cluster's nodes. When one or more of these resources reach
-specific consumption levels, the kubelet can proactively fail one or more pods
+specific consumption levels, the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} can proactively fail one or more pods
```
Comment (sftim) [inline content/en/docs/reference/glossary/node-pressure-eviction.md:21]: We prefer to use a tooltip only for the first relevant use of a term.

**Comment by kirisakow** [comment]
> k8s-ci-robot added the <kbd>do-not-merge/invalid-commit-message</kbd>
> Keywords which can automatically close issues and at(@) or hashtag(#) mentions are not allowed in commit messages.

Oh no :sob:  I can't amend a pushed commit on this fork

**Comment by sftim** [comment]
> Oh no 😭 I can't amend a pushed commit on this fork

Would you like to open a new pull request, with a similar but valid commit?

**Pull request closed without merging**